scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerStarter.java

```diff
@@ -213,6 +220,27 @@ protected static void start(CommandLine commandLine) throws Exception {
         executeStartScripts();
     }
 
+    /**
+     *  Launch boot (i.e., early-starting) microservices (notably, IAM microservice)
+     *
+     */
+    private static void startBootMicroservices(CommandLine commandLine)
+            throws IOException, InterruptedException, ExecutionException {
+
+        // Do nothing if PA_home or the boot microservices path is not specified
+        if (!(PASchedulerProperties.SCHEDULER_BOOT_MICROSERVICES_PATH.isSet() &&
+              PASchedulerProperties.SCHEDULER_HOME.isSet()))
+            return;
+
+        // Acquire paths required to start IAM microservice
+        String pa_home = PASchedulerProperties.getAbsolutePath(System.getProperty(PASchedulerProperties.SCHEDULER_HOME.getKey()));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)